addons/point_of_sale/models/pos_config.py

```diff
@@ -89,7 +89,13 @@ def _default_payment_methods(self):
         return self.env['pos.payment.method'].search([('split_transactions', '=', False), ('company_id', '=', self.env.company.id)])
 
     def _default_pricelist(self):
-        return self.env['product.pricelist'].search([('company_id', 'in', (False, self.env.company.id)), ('currency_id', '=', self.env.company.currency_id.id)], limit=1)
+        pricelist = self.env['product.pricelist'].search([('company_id', 'in', (False, self.env.company.id)), ('currency_id', '=', self.env.company.currency_id.id)], limit=1)
```

Comment:
I am not sure that creating a pricelist in the default is a good idea, because, if you have a company in euro, a default pricelist in euro already exists. And if you want to create a POS in another currency, you probably want to set a real pricelist (the prices won't be the same in euro than in dolars), so creating one is not necessary

Reply:
I put that because I have failing tests in enterprise. I'll try again without this default pricelist.

Reply:
I removed the calculation of default pricelist and the error is not there anymore.